packages/collection/src/frontend.js

```diff
@@ -300,4 +302,43 @@ import {
 		} );
 	}
 
+	function initCarousels() {
+
+	  if ( ! $carousels.length) {
```

Comment:
```js
let $carousels = $('[data-layoutstyle="carousel"]'); 
```
Should be defined here for related code to be in close proximity. Also, by using `$().each` $carousels.length is redundant. You could use:
```js
$('[data-layoutstyle="carousel"]').each( ...
```